standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/TxnHandler.java

```diff
@@ -5086,8 +5088,13 @@ public void performTimeOuts() {
       while(true) {
         stmt = dbConn.createStatement();
         String s = " \"TXN_ID\" FROM \"TXNS\" WHERE \"TXN_STATE\" = " + TxnStatus.OPEN +
```

Comment:
maybe move query to the constants and replace TXN_LAST_HEARTBEAT placeholders here?

Reply:
note: you'll be polluting memory with many String objects every time this method is called.

Reply:
Yes but this needs to go in ASAP so I will skip this extra cleanup task. New changes are not making it worse than it is

Reply:
ok